src/iocore/net/quic/QUICStreamVCAdapter.cc

```diff
@@ -331,7 +331,7 @@ QUICStreamVCAdapter::reenable(VIO * /* vio ATS_UNUSED */)
 bool
 QUICStreamVCAdapter::is_readable()
 {
-  return this->stream().direction() != QUICStreamDirection::SEND && _read_vio.nbytes == _read_vio.ndone;
```

Comment:
This is fine, but didn't you mention `is_writable` as well?

Reply:
I did, but from the tests I made, all streams seems to be removed base on the `is_readable`. Every time a stream is checked for `is_writable` the `write_vio.nbytes` have `INT64_MAX` as value. 

Does it make sense to have `is_writable` performing the same(keeping the direction right direction check) logic than `is_readable`, I mean, as long as they are different, means there are write capabilities on the stream.

Reply:
The `-1` doesn't make sense anyway, nothing sets `-1`. Doing a similar check (`nbytes != ndone`) would make some sense, though it sounds like we have no ways to verify it at the moment. But I don't mind leaving it for now. I was just wondering because you mentioned it on Slack and no change was made for it. There is Unidirectional Stream and it's used for QPACK. We'll perhaps see an issue there once we enabled Dynamic Table.

Reply:
Agreed with `-1`. On my tests, `write_vio.nbytes` was always written after we check for `is_writable`.

I'll change the check for `nbytes != ndone`  just to at least not checking for some value that it is never set.